canonical-web-and-design/maas-ui
# chore(legacy): remove legacy machine list, KVM, RSD, user prefs and settings code

## Done

- Removed legacy machine list, KVM, RSD, user prefs and settings code that have become obsolete from the React migration.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Karura](/HACKING.md#karura)
- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Check the app loads
- Click around different node list/details pages, user preferences and settings and check that everything still runs fine

## Fixes

Fixes #1694
